src/xpanse-api/generated/models/UserOrderableServiceVo.ts

```diff
@@ -8,67 +8,60 @@
 /* tslint:disable */
 /* eslint-disable */
 
+import type { Billing } from './Billing';
+import type { DeployVariable } from './DeployVariable';
+import type { Flavor } from './Flavor';
 import type { Link } from './Link';
-import type { Ocl } from './Ocl';
+import type { Region } from './Region';
 
-export type ServiceTemplateVo = {
+export type UserOrderableServiceVo = {
     /**
-     * ID of the registered service.
+     * The id of the orderable service.
      */
     id: string;
     /**
-     * Name of the registered service.
+     * The category of the orderable service.
+     */
+    category: UserOrderableServiceVo.category;
+    /**
+     * The name of the orderable service.
      */
     name: string;
     /**
-     * Version of the registered service.
+     * The version of the orderable service.
      */
     version: string;
     /**
-     * Csp of the registered service.
+     * The Cloud Service Provider of the orderable service.
      */
-    csp: ServiceTemplateVo.csp;
+    csp: UserOrderableServiceVo.csp;
     /**
-     * Category of the registered service.
+     * The regions of the Cloud Service Provider.
      */
-    category: ServiceTemplateVo.category;
+    regions: Array<Region>;
     /**
-     * Namespace of the user who registered service template.
+     * The description of the orderable service.
      */
-    namespace: string;
-    ocl: Ocl;
+    description: string;
     /**
-     * createTime of the registered service.
+     * The icon of the orderable service.
      */
-    createTime: string;
+    icon: string;
     /**
-     * Last updateTime of the registered service.
+     * The variables for the deployment, which will be passed to the deployer.
      */
-    lastModifiedTime: string;
+    variables: Array<DeployVariable>;
     /**
-     * State of service.
+     * The flavors of the orderable service.
      */
-    serviceRegistrationState: ServiceTemplateVo.serviceRegistrationState;
+    flavors: Array<Flavor>;
```

Comment:
@baixinsui should we generate the client again?

Reply:
The field 'price' in  the class Flavor is necessary, so there is no need to regenerate it